src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -378,9 +384,9 @@ def body(self, data, data_type, **kwargs):
             # Be sure to pass a DeploymentProperties
             template = data.properties.template
             if template:
-                data.properties.template = None
```

Comment:
@Juliehzl I made this change to make --handle-extended-json-format parameter work. Let me know if you see an issue.